### PR TITLE
Feature/#1861 miro integration

### PR DESCRIPTION
**Pull request by PAKovalev**

### CODE REVIEW

File: src/api/repositories/integrationRepository/integrationRepository.api.ts [hunk 1]
```diff
@@ -40,6 +40,8 @@ export const disableIntegration: RequestInterface<
 > = (options) => {
   const {spaceId, integrationType, data, ...rest} = options;
 
+  alert(1);
```
Comment (aminbksi) [inline src/api/repositories/integrationRepository/integrationRepository.api.ts]: what's this for?

File: src/api/repositories/integrationRepository/integrationRepository.api.ts [hunk 2]
```diff
@@ -24,24 +25,28 @@ export const fetchIntegration: RequestInterface<
...
+  DisableMiroIntegrationResponse
 > = (options) => {
-  const {spaceId, integrationType, data, ...rest} = options;
+  const {spaceId, ...rest} = options;
 
-  alert(1);
+  const data = {
+    spaceId,
+    integrationType: IntegrationTypeEnum.MIRO,
+    data: {id: '', name: '', description: '', viewLink: '', accessLink: '', embedHtml: ''}
```
Comment (aminbksi) [inline src/api/repositories/integrationRepository/integrationRepository.api.ts:47]: why are you sending data if they are null?!

**Review by aminbksi — APPROVED** [review]

File: src/api/repositories/integrationRepository/integrationRepository.api.ts (hunk 2 above)
Comment (PAKovalev) [reply src/api/repositories/integrationRepository/integrationRepository.api.ts:47]: it is BE implementation.

Comment (PAKovalev) [reply src/api/repositories/integrationRepository/integrationRepository.api.ts:47]: it doesn't work if we don't send empty object

**Review by Daboo1998 — CHANGES_REQUESTED** [review]

File: src/core/models/Integration/models/IntegrationUser/IntegrationUser.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import {types} from 'mobx-state-tree';
+
+import {UUIDModel} from 'core/models';
+
+import {IntegrationUserData} from './models';
+
+const IntegrationUser = types.model('IntegrationUser', {
+  userId: UUIDModel,
+  spaceId: types.frozen<Buffer>(),
```
[the remark this answers is not in the document]
Comment (PAKovalev) [reply src/core/models/Integration/models/IntegrationUser/IntegrationUser.ts:9]: It is buffer, not UUIDModel

File: src/api/repositories/integrationRepository/integrationRepository.api.ts (hunk 2 above)
Comment (PAKovalev) [reply src/api/repositories/integrationRepository/integrationRepository.api.ts:47]: not critical for now. We can broke something in other integration.

File: src/core/models/Integration/models/IntegrationUser/IntegrationUser.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  integrationTypeId: types.frozen<Buffer>(),
```
[the remark this answers is not in the document]
Comment (PAKovalev) [reply src/core/models/Integration/models/IntegrationUser/IntegrationUser.ts:10]: it is just buffer, not UUID

File: src/ui-kit/templates/SpaceTopBar/components/RightSection/RightSection.tsx [hunk 5]
```diff
@@ -3,45 +3,44 @@ import {observer} from 'mobx-react-lite';
...
-  spaceId?: string;
-  favoriteStore: FavoriteStoreInterface;
+  spaceId: string;
+  editSpaceHidden?: boolean;
+  isSpaceFavorite: boolean;
+  toggleIsSpaceFavorite: (spaceId: string) => void;
 }
 
-const TopBarActions: FC<PropsInterface> = ({isAdmin, favoriteStore, spaceId}) => {
+const RightSection: FC<PropsInterface> = ({
```
[the remark this answers is not in the document]
Comment (PAKovalev) [reply src/ui-kit/templates/SpaceTopBar/components/RightSection/RightSection.tsx:23]: This component under SpaceTopBar. Not reusable component.

[the remark this answers is not in the document]
Comment (Daboo1998) [reply src/ui-kit/templates/SpaceTopBar/components/RightSection/RightSection.tsx:23]: Then I would suggest it not to be able to be imported straight from 'ui-kit'

Cause now you can do `import {RightSection} from 'ui-kit';` if its only used as you describe then I guess it shouldn't be a case.

File: src/core/models/Integration/models/IntegrationUser/IntegrationUser.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (PAKovalev) [reply src/core/models/Integration/models/IntegrationUser/IntegrationUser.ts:9]: Not at this moment. Just buffer

File: src/ui-kit/templates/SpaceTopBar/components/RightSection/RightSection.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (PAKovalev) [reply src/ui-kit/templates/SpaceTopBar/components/RightSection/RightSection.tsx:23]: No, Did you check code?

`import {RightSection} from './components';`

**Review by Daboo1998 — APPROVED** [review]

**Pull request merged**